docs/farming-&-staking/farming/advanced-cli/cli-install.mdx

```diff
@@ -63,3 +63,12 @@ We will download two files for your respective operating system.
 <SourcePage />
 </TabItem>
 </Tabs>
+
+:::note Learn More About Syncing, Plotting & Farming
+
+If you would like to learn more about how the Syncing, Plotting, & Farming process works on the Subspace Network, please refer to our Subnomicon links below.
```

Comment:
I'm not sure whether the process here should be singular, or not 🤔 Would likely use "processes" as it refers to multiple different instances.

Reply:
Fixed with latest commit [65b91bd](https://github.com/subspace/subspace-docs/pull/426/commits/65b91bd957b150c423dca0683b25fae3f558b31c)